Runtime/src/index.ts

```diff
@@ -146,6 +159,26 @@ export class SwiftRuntime {
             throw new Error("The versions of JavaScriptKit are incompatible.");
         }
     }
+    get closureHeap(): SwiftClosureHeap | null {
+        if (this._closureHeap)
+            return this._closureHeap;
+        if (!this.instance)
+            throw new Error("WebAssembly instance is not set yet");
+
+        const exports = (this.instance
+            .exports as any) as SwiftRuntimeExportedFunctions;
+        const features = exports.swjs_library_features();
+        const librarySupportsWeakRef = (features & LibraryFeatures.WeakRefs) != 0;
+        if (librarySupportsWeakRef) {
+            if (typeof FinalizationRegistry !== "undefined") {
+                this._closureHeap = new SwiftClosureHeap(exports);
+		return this._closureHeap;
```

Comment:
check your whitespace here — it looks like tabs got used instead of spaces

Reply:
I forgot to apply prettier 😅